src/utils/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+file (GLOB siminusminus_utils_sources
+		"*.cpp")
```

Comment:
**NERVER NERVER NEVER NERVER GLOB CMAKE SOURCES**. Explicitly list the sources in the target